packages/constants/src/index.js

```diff
@@ -33,6 +33,7 @@ export const SPACING = {
   SCALE_6: '60px',
 };
 
+// TODO: figure out how to optionally include locally installed font, e.g. "GDS Transport Website"
```

Comment:
Could it not be part of the font stack?

Edit; ah oki it already is, but does not work if you have the font locally installed?

Reply:
only if you have it installed as 'nta' - the one I have is called something else locally